### PR TITLE
Support arm64-v8a

### PR DESCRIPTION
**Pull request by yamashi**

### CODE REVIEW

File: packages/d/dotnet/xmake.lua [hunk 1]
```diff
@@ -15,7 +15,7 @@ package("dotnet")
         if os.arch() == "x86_64" then
             set_urls("https://download.visualstudio.microsoft.com/download/pr/bda88810-e1a6-4cf0-8139-7fd7fe7b2c7a/7a9ffa3e12e5f1c3d8b640e326c1eb14/dotnet-sdk-7.0.202-linux-x64.tar.gz")
             add_versions("7.0.202", "405f15e437582be260460f48eda9dfe613fd87b2557667f20d6ecfa34b09c221")
-        elseif os.arch() == "arm64" then
+        elseif os.arch() == "arm64" or os.arch() == "arm64-v8a" then
```
Comment (SirLynix) [inline packages/d/dotnet/xmake.lua:18]: `elseif is_arch("arm64", "arm64-v8a")`

File: packages/d/dotnet/xmake.lua [hunk 2]
```diff
@@ -38,7 +38,8 @@ package("dotnet")
             os.cp(path.join(out_path, "nethost.dll"), package:installdir("bin"))
             os.cp(path.join(out_path, "nethost.lib"), package:installdir("lib"))
         elseif package:is_plat("linux") then
-            out_path = path.join(out_path, "Microsoft.NETCore.App.Host.linux-" .. (package:is_arch("arm64") and "arm64" or "x64"), "7.0.4", "runtimes", "linux-" .. (package:is_arch("arm64") and "arm64" or "x64"), "native")
+            local is_arm = package:is_arch("arm64") or package:is_arch("arm64-v8a")
```
Comment (SirLynix) [inline packages/d/dotnet/xmake.lua:41]: `package:is_arch("arm64", "arm64-v8a")`

File: packages/d/dotnet/xmake.lua [hunk 3]
```diff
@@ -15,7 +15,7 @@ package("dotnet")
         if os.arch() == "x86_64" then
             set_urls("https://download.visualstudio.microsoft.com/download/pr/bda88810-e1a6-4cf0-8139-7fd7fe7b2c7a/7a9ffa3e12e5f1c3d8b640e326c1eb14/dotnet-sdk-7.0.202-linux-x64.tar.gz")
             add_versions("7.0.202", "405f15e437582be260460f48eda9dfe613fd87b2557667f20d6ecfa34b09c221")
-        elseif os.arch() == "arm64" or os.arch() == "arm64-v8a" then
+        elseif is_arch("arm64", "arm64-v8a") then
```
Comment (waruqi) [inline packages/d/dotnet/xmake.lua:18]: and other if branches

Comment (yamashi) [reply packages/d/dotnet/xmake.lua:18]: I don't understand what you mean?

Comment (yamashi) [reply packages/d/dotnet/xmake.lua:18]: oh using is_arch?

**Pull request merged**